<launch>
  <!--
    <node pkg="planning" type="dev_global_route.py" name="DevGlobalRoute" output="screen">
           <param name="from_txt" value="True" />
           <param name="sampling_resolution" value="75.0" />
           <param name="routes" value="/opt/leaderboard/data/routes_devtest.xml" />
           <param name="global_route_txt" value="/code/planning/global_planner/src/global_route.txt" />
           <param name="role_name" value="hero" />
    </node>-->
    <node pkg="planning" type="global_planner.py" name="PrePlanner" output="screen">
           <param name="role_name" value="hero" />
           <param name="control_loop_rate" value="1" />
           <param name="distance_spawn_to_first_wp" value="100" />
    </node>
</launch>
